app/templates/submit.hbs

```diff
@@ -14,9 +14,22 @@
     <div class="container">
         <div class="row m-t-lg">
             <div class="col-xs-12 col-md-10 col-md-offset-1 ">
-                <p>{{if editMode (t "submit.body.p_edit") (t "submit.body.p_add" name=(if theme.isProvider theme.provider.name (t "global.brand_name")))}}</p>
+                <p>{{if editMode (t "submit.body.p_edit") (t (if theme.isProvider "submit.body.p_add_provider" "submit.body.p_add") name=(if theme.isProvider theme.provider.name (t "global.brand_name")))}}</p>
                 {{#cp-panels accordion=true}}
-                    {{#with _names.[0] as |name|}}
+                    {{#unless editMode}}
```

Comment:
Thanks @abought! I started using truth helpers elsewhere and forgot to go back and refactor this.